src/__tests__/Radio.js

```diff
@@ -7,7 +7,7 @@ describe('Radio', () => {
     const json = renderer
       .create(
         <ThemeProvider>
-          <Radio checked />
+          <Radio checked id="radio-input" />
```

Comment:
Trivial, if the PR removes the `id` attribute, can the test code also remove the `id` attribute?

Reply:
I'm guessing this was added to compensate for the snapshot changes, but I think it's okay to update the snapshots with this PR by running `npm t -- -u`